src/main/java/org/jabref/gui/maintable/MainTable.java

```diff
@@ -322,6 +368,14 @@ private void setupKeyBindings(KeyBindingRepository keyBindings) {
                         deleteAction.execute();
                         event.consume();
                         break;
+                    case SCROLL_TO_NEXT_RANK:
+                        scrollToRank(1);
+                        event.consume();
+                        break;
+                    case SCROLL_TO_PREVIOUS_RANK:
+                        scrollToRank(-1);
+                        event.consume();
+                        break;
```

Comment:
RANK as ints is meaningless. scrollToRank should with constants. Maybe introduce method scrollToNextRank/scrollToPreviousRank or sthg.

Reply:
https://github.com/JabRef/jabref/pull/11510#discussion_r1700965442